package.json

```diff
@@ -12,14 +12,13 @@
     "format": "prettier . --write --cache --cache-location=node_modules/.cache/.prettiercache --log-level=warn",
     "format:check": "prettier . --check --cache --cache-location=node_modules/.cache/.prettiercache",
     "lint": "turbo run lint",
-    "publint": "turbo run publint --concurrency=1",
     "are-the-types-wrong": "turbo run are-the-types-wrong",
     "manypkg": "manypkg check",
     "typesync": "typesync --dry=fail",
     "typecheck": "turbo run typecheck",
-    "knip": "knip",
+    "knip": "knip --treat-config-hints-as-errors",
     "validate-config": "turbo run validate-config",
-    "merge-checks": "pnpm build && turbo run publint manypkg typesync knip typecheck lint validate-config are-the-types-wrong && pnpm check-deduped-deps",
```

Comment:
Yay, this hack is done!

Reply:
this wasn't a hack. This was here to check if the build was passing before doing the lints and the tests.

Reply:
Build is called as part of merge-checks (except for docs), putting it before is antithetical to turbo (i.e., the tasks are misconfigured), and slower.

If you'd like it reverted, I will, but I don't see why this would be the desired behavior.